de_templates/ingest/Ingest_AzureSql_SourceName/data_factory/adf_pipelines.tf.jinja

```diff
@@ -0,0 +1,14 @@
+resource "azurerm_resource_group_template_deployment" "pipeline_Ingest_AzureSql_{{ source_name }}_DQ" {
+  name                = "pipeline_Ingest_AzureSql_{{ source_name }}_DQ"
+  resource_group_name = var.data_factory_resource_group_name
+  deployment_mode     = var.arm_deployment_mode
+  parameters_content = jsonencode({
+    "factoryName" = {
+      value = data.azurerm_data_factory.factory.name
+    }
+  })
+  template_content = file("${path.module}/pipelines/Ingest_AzureSql_DQ.json")
```

Comment:
Pipeline file name will need to be dynamic.

Reply:
done